Apsitvarkom.UnitTests/Models/Mapping/CleaningEventMappingTests.cs

```diff
@@ -58,5 +58,33 @@ public void CleaningEventToCleaningEventResponse()
             Assert.That(response.PollutedLocationId, Is.EqualTo(businessLogicObject.PollutedLocationId));
         });
     }
+
+    [Test]
+    public void CleaningEventCreateRequestToCleaningEvent()
+    {
+        var businessLogicObject = new CleaningEvent
+        {
+            Id = Guid.NewGuid(),
+            StartTime = new DateTime(2022, 9, 16, 21, 43, 31).ToUniversalTime(),
+            Notes = "Hello world",
+            PollutedLocationId = Guid.NewGuid()
+        };
+
+        var createModel = new CleaningEventCreateRequest
+        {
+            StartTime = new DateTime(2022, 9, 16, 21, 43, 31).ToUniversalTime(),
+            Notes = "Hello world",
+            PollutedLocationId = Guid.NewGuid()
+        };
+
+        var response = _mapper.Map<CleaningEventCreateRequest, CleaningEvent>(createModel, businessLogicObject);
+        Assert.Multiple(() =>
```

Comment:
usually the structure of unit tests are:
- Arrange
- Act
- Assert

at least in my workspace, we usually try to separate all of these parts with new lines, if that's possible and/or logical. we could add a new line here as well.
```suggestion
        var response = _mapper.Map<CleaningEventCreateRequest, CleaningEvent>(createModel, businessLogicObject);
        
        Assert.Multiple(() =>
```